pandas/core/frame.py

```diff
@@ -11176,6 +11177,10 @@ def quantile(
             * higher: `j`.
             * nearest: `i` or `j` whichever is nearest.
             * midpoint: (`i` + `j`) / 2.
+        method : {'single', 'table'}, default 'single'
+            Whether to compute quantiles per-column ('single') or over all columns
+            ('table'). When 'table', the only allowed interpolation methods are
+            'nearest', 'lower', and 'higher'.
```

Comment:
Just to confirm, the allowed interpolation methods are based on `np.quantile` interpolation support?